src/components/composites/Header/index.js

```diff
@@ -39,87 +44,99 @@ export default function Header() {
   };
 
   return (
-    <HideOnScroll>
-      <Box sx={{ flexGrow: 1 }}>
-        <AppBar position="static">
-          <Toolbar variant="dense">
-            <Box
-              sx={{
-                flexGrow: 1,
-                display: "flex",
-                flexDirection: "row",
-                alignItems: "center",
-              }}
-            >
-              <Typography variant="h4" sx={{ fontWeight: "bold" }}>
-                🏍️ tMX
-              </Typography>
-              {/* <img
+    <>
+      {!showHeader ? (
```

Comment:
change the var name - you are showing the headers either way, it should be called something like showMobile